src/cmds/scripts/pbs_snapshot

```diff
@@ -53,44 +53,48 @@ if [ ! -d  ${ptl_prefix_lib} ]; then
     fi
 fi
 
+conf="${PBS_CONF_FILE:-/etc/pbs.conf}"
```

Comment:
If you do the change that I suggested, then you won't have to move this block of code